clients/admin-ui/src/features/config-wizard/PrivacyDeclarationForm.tsx

```diff
@@ -0,0 +1,387 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Button,
+  FormLabel,
+  Heading,
+  Stack,
+  Text,
+  Tooltip,
+  useToast,
+} from "@fidesui/react";
+import { SerializedError } from "@reduxjs/toolkit";
+import { FetchBaseQueryError } from "@reduxjs/toolkit/dist/query/fetchBaseQuery";
+import { Form, Formik } from "formik";
+import type { NextPage } from "next";
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import { AddIcon, QuestionIcon } from "~/features/common/Icon";
+import {
+  selectDataQualifier,
+  setDataQualifier,
+  useGetDataQualifierQuery,
+} from "~/features/data-qualifier/data-qualifier.slice";
+import {
+  selectDataSubjects,
+  setDataSubjects,
+  useGetAllDataSubjectsQuery,
+} from "~/features/data-subjects/data-subject.slice";
+import {
+  selectDataUse,
+  setDataUse,
+  useGetDataUseQuery,
+} from "~/features/data-use/data-use.slice";
+import {
+  selectDataCategories,
+  setDataCategories,
+  useGetAllDataCategoriesQuery,
+} from "~/features/taxonomy/data-categories.slice";
+
+import {
+  CustomMultiSelect,
+  CustomSelect,
+  CustomTextInput,
+} from "../common/form/inputs";
+import { isErrorWithDetail, isErrorWithDetailArray } from "../common/helpers";
+import {
+  useGetSystemByFidesKeyQuery,
+  useUpdateSystemMutation,
+} from "../system/system.slice";
+
+interface PrivacyDeclaration {
+  name: string;
+  data_categories: string[];
+  data_use: string;
+  data_qualifier: string;
+  data_subjects: string[];
+}
+
+type FormValues = Partial<PrivacyDeclaration>;
+
+const PrivacyDeclarationForm: NextPage<{
+  handleChangeReviewStep: Function;
+  handleCancelSetup: Function;
+}> = ({ handleCancelSetup, handleChangeReviewStep }) => {
+  const dispatch = useDispatch();
+  const toast = useToast();
+  const [formDeclarations, setFormDeclarations] = useState<any>([]);
+  const [updateSystem] = useUpdateSystemMutation();
+  const [isLoading, setIsLoading] = useState(false);
+  // TODO FUTURE: Need a way to check for an existing fides key from the start of the wizard
+  // not just use this default
+  const { data: existingSystem } = useGetSystemByFidesKeyQuery(
+    "default_organization"
+  );
```

Comment:
Why is `default_organization` used as the key to query for systems? That key should only map to an organization right?

Reply:
@earmenda It's being used as the default `fides_key` at the moment in the UI. I was told to use this as the `fides_key` for now in the code.

Reply:
`default_organization` should only be used for the expected default organization, there isn't a default system with the fides key of `default_organization`

Reply:
@ThomasLaPiana - Can we talk through this, maybe after our 1-1 tomorrow? From my understanding, at this point in development, I should be using `default_organization` as the `fides_key` for now.  I am creating systems for this organization in the wizard. When I need to recall information about these systems, I am calling the API and getting any existing system information using the fides_key as described in the API (GET `/system/{fides_key}`). This call is grabbing any existing system information from the organization using the fides_key. The purpose of grabbing any existing information about a system is because a user can save progress and also go back and edit information as they register their information in the wizard.

Let me know if I'm misunderstanding something about the API or if I need to get existing system information another way. Happy to discuss.

Reply:
@LKCSmith totally correct at the end there. a system's `fides_key` is unique to that system, it doesn't point to the parent org. You're right in that `organization_fides_key` is the `fides_key` that points to the parent org 💯